pkg/ebpf/bpf_arm64_bpfel.go

```diff
@@ -75,35 +87,35 @@ type BpfFilterValueT struct {
 type BpfFlowId BpfFlowIdT
 
 type BpfFlowIdT struct {
-	EthProtocol       uint16
 	Direction         uint8
-	SrcMac            [6]uint8
-	DstMac            [6]uint8
 	SrcIp             [16]uint8
 	DstIp             [16]uint8
+	_                 [1]byte
 	SrcPort           uint16
 	DstPort           uint16
 	TransportProtocol uint8
 	IcmpType          uint8
 	IcmpCode          uint8
+	_                 [3]byte
```

Comment:
this example of what removing packed will cause BTW we could avoid so gaps by moving the fields around if u move direction to the bottom might help this struct ?

Reply:
so in this PR I haven't care of these little optimizations because in the next PR I still move stuff around and [there](https://github.com/netobserv/netobserv-ebpf-agent/pull/470/commits/f0d743c47c838c5666b6d9cbd5fc37069dbd7998#diff-dbaedeecfc6f45e9f0f7521e5146144ca41b968d50b0acad82393f70d2b2e20f) I'm caring about these optimizations